OpenRA.Mods.D2k/Traits/Carryall.cs

```diff
@@ -103,6 +103,9 @@ void FindCarryableForTransport()
 					if (!trait.WantsTransport)
 						return false;
 
+					if (actor.IsIdle)
```

Comment:
More awesome behavior from the Carryall code.
 - We don't want carryalls to pick up units that are idle. Those obviously have nothing to do and nowhere to go.
 - The `WantsTransport` flag isn't cleared when the related order is cancelled, so it can get picked up at a later point, when it no longer needs transportation.